docs/get_started/get_started_linux.md

```diff
@@ -139,35 +139,6 @@ Throughput: 375.3339402 FPS
 
 </details>
 
-### Inference Pipeline Demo Script
-The `demo_security_barrier_camera` uses vehicle recognition in which vehicle attributes build on each other to narrow in on a specific attribute.
-
-The script:
-1. Downloads three pre-trained model IRs.
-2. Builds the Security Barrier Camera Demo application.
-3. Runs the application with the downloaded models and the `car_1.bmp` image from the `demo` directory to show an inference pipeline. 
-
-This application:
-
-1. Identifies an object identified as a vehicle. 
-2. Uses the vehicle identification as input to the second model, which identifies specific vehicle attributes, including the license plate.
-3. Uses the the license plate as input to the third model, which recognizes specific characters in the license plate.
-
-<details>
-    <summary><strong>Click for an example of Running the Pipeline demo script</strong></summary>
-    
-To run the script performing inference on Intel® Processor Graphics:
-
-```sh
-./demo_security_barrier_camera.sh -d GPU
-```
-
-When the verification script completes, you see an image that displays the resulting frame with detections rendered as bounding boxes, and text:
-
-![](../img/inference_pipeline_script_lnx.png)
-
-</details>
-
 ### Benchmark Demo Script
```

Comment:
```suggestion
### Benchmark Sample Script
```

I would prefer to call IE Samples as samples and OMZ Demos as demos

Reply:
but the scripts itself are named `demo_***`. Should we rename as well?

Reply:
@ilya-lavrenov My vote would be for renaming. It seems that names like:
sample_benchmark_app.[bat/sh]
sample_speech_recognition.[bat/sh]
sample_squeezenet_download_convert_run.[bat/sh]
or even
go_sample_benchmark_app.[bat/sh]
go_sample_speech_recognition.[bat/sh]
go_sample_squeezenet.[bat/sh] (go_sample_squeezenet_end2end.[bat/sh] ?)
looks good enough and provide correct positioning of those.

Reply:
Ok, I agree with renaming. Suggest:
```sh
run_sample_benchmark_app.[bat/sh]
run_sample_speech_recognition.[bat/sh]
run_sample_squeezenet.[bat/sh]
```

Reply:
@dpigasin could you please rename to suggested: run_sample_... ?

Reply:
@dpigasin well, I did the required changes

@ilya-lavrenov FYI